ecosystem/sep-0010.md

```diff
@@ -18,12 +18,21 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
-The authentication flow is as follows:
+It involves three parties:
+- A home domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` value.
+- A web auth endpoint providing the GET and POST operations discussed in this document.
+- A client who will authenticate with a Stellar account.
+
+The discovery flow is as follows:
+
+1. The client retrieves the [SEP-1 stellar.toml](sep-0001.md) from the home domain. The home domain hosts the stellar.toml.
+1. The client looks up the `WEB_AUTH_ENDPOINT` value from the [SEP-1 stellar.toml](sep-0001.md).
```

Comment:
```suggestion
1. The client looks up and notes the `WEB_AUTH_ENDPOINT` and `SIGNING_KEY` values from the [SEP-1 stellar.toml](sep-0001.md). If either of those values is missing, client must not continue with the authentication.
```

Reply:
I think adding this requirement is probably not going to change clients/servers. If a `WEB_AUTH_ENDPOINT` is not present the following steps are not possible so we do not need to call it out. If a `SIGNING_KEY` is not present and a client chose to continue the rest of the flow is unlikely to fail. I think the signing key verification being a hard requirement is something we need to revisit and so I'm hesitant to write it into more areas of the SEP unnecessarily.

Reply:
This change was primarily to localise in a single place all the data that client has to obtain about the server and emphasise that all this happens during discovery flow, so that we can easily reference it from key rotation support condition below (re-run the discover on failed signing key validation). I added the obvious halt condition just for the sake of completeness. I'm not sure I understand your concerns regarding hard requirement verification of the signing key, lets discuss it on keybase.

Reply:
fdcfa0c b37dac3